### PR TITLE
fix: disable the Email username check for Mailtrap

### PR DESCRIPTION
**Pull request by longxu0509**
For: https://github.com/casdoor/casdoor/issues/1516

### CODE REVIEW

File: send.go [hunk 1]
```diff
@@ -34,10 +32,8 @@ func (f SendFunc) Send(from string, to []string, msg io.WriterTo) error {
 
 // Send sends emails using the given Sender.
 func Send(s Sender, msg ...*Message) error {
-	for i, m := range msg {
-		if err := send(s, m); err != nil {
-			return fmt.Errorf("gomail: could not send email %d: %v", i+1, err)
-		}
+	for _, m := range msg {
```
Comment (hsluoyz) [inline send.go]: Don't skip the errors returned by SMS providers!

File: send.go [hunk 2]
```diff
@@ -108,9 +104,6 @@ func addAddress(list []string, addr string) []string {
 }
 
 func parseAddress(field string) (string, error) {
-	addr, err := mail.ParseAddress(field)
-	if err != nil {
-		return "", fmt.Errorf("gomail: invalid address %q: %v", field, err)
-	}
-	return addr.Address, nil
+
+	return field, nil
```
Comment (hsluoyz) [inline send.go]: Only need to skip the "@" check, all other errors should be kept!

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: send.go (hunk 1 above)
Comment (longxu0509) [reply send.go]: Okay，Thank you for your code review comment. Now I don't  skip the errors returned by SMS providers!

File: send.go (hunk 2 above)
Comment (longxu0509) [reply send.go]: Done

**Pull request closed without merging**